src/main/java/mods/railcraft/world/entity/cart/locomotives/ElectricLocomotiveEntity.java

```diff
@@ -50,8 +57,8 @@
 
   private final IInventory ticketInventory =
       new InventoryMapper(this, SLOT_TICKET, 2).ignoreItemChecks();
-  private final LazyOptional<IBatteryCart> cartBattery =
```

Comment:
This was here to keep the `LazyOptional` instance cached instead of creating a new one every time `getCapability` is called.